sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/tracing/TraceTokenConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.tracing;
+
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@AutoConfiguration
+public class TraceTokenConfiguration implements WebMvcConfigurer {
+
+  @Bean
+  public TraceTokenRequestInterceptor traceTokenInterceptor() {
+    return new TraceTokenRequestInterceptor();
+  }
+
+  @Override
+  public void addInterceptors(InterceptorRegistry registry) {
+    registry.addInterceptor(traceTokenInterceptor());
```

Comment:
why don't we need to add the `HttpRequestLoggingInterceptor` here?

Reply:
I created separate Configurations, one for HttpRequestLogging and one for the TraceToken